settings.py

```diff
@@ -39,6 +39,7 @@
 # the number of seconds chaosbot should sleep between polling for ready prs
 PULL_REQUEST_POLLING_INTERVAL_SECONDS = 30
 ISSUE_COMMENT_POLLING_INTERVAL_SECONDS = 60 * 10  # 10 min window on polling comments
+ISSUE_CLOSE_STALE_INTERVAL_SECONDS = 60 * 10  # 10 min window on polling issues
```

Comment:
🤔 Is there an option to `run right now` and repeat every n seconds/minutes/hours?